src/main/resources/public/js/src/launches/suiteLevel/SuiteControlView.js

```diff
@@ -46,7 +46,7 @@ define(function (require, exports, module) {
         computeds: {
             validateForHistoryBtn: function(){
                 var interrupted = config.launchStatus.interrupted,
-                    showBtn = this.parentModel.get('status') !== interrupted && this.launchModel.get('status') !== interrupted && !_.isEmpty(this.collectionItems.models);
+                    showBtn = this.launchModel.get('status') !== interrupted && !this.collectionItems.validateForAllCases() && !_.isEmpty(this.collectionItems.models);
```

Comment:
Each variable must be declared separately.
https://github.com/airbnb/javascript#variables